dell/xps/17-9700/nvidia/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, pkgs, ... }:
+
+{
+  imports = [
+    ../../../../common/cpu/intel
```

Comment:
common/gpu/nvidia could also be added here, this has a part of the options below already included